ui/src/Pages/Home/CheckoutPage.tsx

```diff
@@ -0,0 +1,181 @@
+import React, { useState } from "react";
+import { Container, Grid, Typography, Button, Paper, Divider, FormControl, InputLabel, Select, MenuItem, TextField, Dialog, DialogTitle, DialogContent, DialogActions } from "@mui/material";
+import { Link as RouterLink } from "react-router-dom";
+
+function CheckoutPage() {
+    const subtotal = 5000; // Replace with actual subtotal calculation
+    const delivery = 20; // Replace with actual delivery fee calculation
+    const tax = 20; // Replace with actual tax calculation
+    const total = subtotal + delivery + tax; // Total calculation
```

Comment:
you could move this to another file in a folder called test data under ui/src. Import all of these numbers in this file and use them wherever you need